infra/app/database.bicep

```
@@ -71,6 +79,15 @@ var containers = [
         }
       ]
     }
+    fullTextPolicy: {
+      defaultLanguage: 'en-US',
+      fullTextPaths: [
+        {
+          path: '/description',
```

Comma throwing syntax error

```suggestion
          path: '/description'
```